client/views/changelog.tpl

```diff
@@ -0,0 +1,20 @@
+{{#if latest}}
+<a href="{{latest.url}}" target="_blank" rel="noopener" class="changelog-version changelog-version-new">
+	The Lounge <b>{{latest.version}}</b>{{#if latest.prerelease}} (pre-release){{/if}} is now available.
+	Click to view details on GitHub.
+</a>
+{{else if current.changelog}}
+<div class="changelog-version">
+	The Lounge is up to date!
+</div>
+{{/if}}
+
+<h1 class="title">Changelog for {{current.version}}</h1>
```

Comment:
I don't think there is a way to go back to the Help window yet:

- The back/forward button does not work
- What about adding a "Back to the Help page" button here?

Reply:
I made the back/forward buttons sort of work. You can go back, forward, but next back won't work, don't know why.

Reply:
Uh, that's odd.
Side-node: There is no... back button on iOS... 😩
I'm glad we have a &times; button on the image preview.